src/Services/AuraService.ts

```diff
@@ -0,0 +1,172 @@
+import {
+  Epoch,
+  EpochOverview,
+  OverviewResponse,
+} from "@/Apps/LlamaAirforce/Pages/Bribes/Models";
+import {
+  AuraConstants,
+  getLatestAuraRound,
+} from "@/Apps/LlamaAirforce/Pages/Bribes/Util/AuraHelper";
+import ServiceBase from "@/Services/ServiceBase";
+import { flattenDeep } from "lodash";
+
+type GaugeResponse = {
+  data: GaugeVote[];
+};
+
+type GaugeVote = {
+  proposal: string;
+  proposalHash: string;
+  title: string;
+  proposalDeadline: number;
+  totalValue: number;
+  maxTotalValue: number;
+  voteCount: number;
+  valuePerVote: number;
+  maxValuePerVote: number;
+  bribes: Bribe[];
+  index: number;
+};
+
+type Bribe = {
+  token: string;
+  symbol: string;
+  decimals: number;
+  value: number;
+  maxValue: number;
+  amount: number;
+  maxTokensPerVote: number;
+  briber: string;
+  periodIndex: number;
+  chainId: number;
+};
+
+const { HH_API_URL, LA_API_URL, START_ROUND, START_DATE, BIWEEKLY } =
+  AuraConstants;
+
+export default class AuraService extends ServiceBase {
```

Comment:
Given this class is only used in `\src\Apps\LlamaAirforce\Pages\Bribes` it's probably best to move it next to `AuraBribeService.ts`.